src/reanimated2/mappers.ts

```diff
@@ -131,17 +131,21 @@ function createMapperRegistry() {
     }
   }
 
-  function extractInputs(
-    inputs: any,
-    resultArray: SharedValue<any>[]
-  ): SharedValue<any>[] {
+  function isBaseObject(obj: unknown): obj is object {
```

Comment:
```suggestion
  function isPlainObject(obj: unknown): obj is object {
```

btw I thought we already have such function somewhere

Reply:
[98d3164](https://github.com/software-mansion/react-native-reanimated/pull/5363/commits/98d3164603d97e5f6800c266e7d7f79507de2a9e)